src/include/metadata/iceberg_table_metadata.hpp

```diff
@@ -48,13 +48,16 @@ struct IcebergTableMetadata {
 	int32_t iceberg_version;
 	int32_t current_schema_id;
 	int32_t default_spec_id;
+	// int32_t default_sort_order_id;
 
 	bool has_current_snapshot = false;
 	int64_t current_snapshot_id;
 	int64_t last_sequence_number;
 
 	//! partition_spec_id -> partition spec
 	unordered_map<int32_t, IcebergPartitionSpec> partition_specs;
+	//! sort_order_id -> sort order
```

Comment:
Is this supposed to be commented out?

Reply:
I'll delete it. Eventually I think we will need it, but not until we support altering tables/partitions/sort orders